website/payments/templates/admin/payments/change_form.html

```diff
@@ -11,6 +11,18 @@
     <script type="text/javascript" src="{% static 'admin/payments/js/payments.js' %}"></script>
 {% endblock %}
 
+{% block after_field_sets %}
+    {% if batch %}
+    <div class="submit-row payments-row">
+        {% if not processed %}
```

Comment:
```suggestion
        {% if not batch.processed %}
```
I mentioned this value in the view class that uses this template.